frontend/src/scenes/onboarding/OnboardingStep.tsx

```diff
@@ -73,6 +75,7 @@ export const OnboardingStep = ({
                 <p>{subtitle}</p>
                 {children}
                 <div className="mt-8 flex justify-end gap-x-2">
+                    {helpButton && helpButton}
```

Comment:
If we know this is a button, it seems like the prop should be `helpButtonAction` and if it exists then render the correct button here. If it has to be passed in every time the button on one page could not match the button on another.